.github/workflows/scripts/changelog-release-helper.mjs

```diff
@@ -69,6 +104,18 @@ export function validateVersion(newVersion) {
  * @param {string} newVersion
  */
 export function updateChangelog(newVersion) {
+  // Skip the entire function if the release version is internal eg: 5.1.0-internal.0
```

Comment:
Nice, so my understanding is:

If we're running `internal` releases, once we sort the `base` thing, we can do as many as we like.

Then when we want to switch to `beta` releases, all the checks will pass because we haven't actually updated the changelog for the internal releases and `[current-release]-beta.0` will be valid?

Reply:
That's the theory yes!